### DISCUSSION TITLE
如何在主题pjax函数里加window.TWIKOO_MAGIC_PATH = ''

### DISCUSSION THREAD

**Discussion by ghost** [HEIGE-PCloud/DoIt#345] [Q&A] [answered]
我在适配 中英文 的页面，并且都开启评论功能，同时也有其他特定的页面开启评论。
但是，**没有发表过评论的页面，会将其他页面中的评论内容带过来，导致评论内容的错乱**。（即，A页面，会显示B页面的评论内容）
我使用的是评论系统是 Twikoo。

请问我如何 “在主题pjax函数里加window.TWIKOO_MAGIC_PATH = '' ” ？

问题原文：https://github.com/imaegoo/twikoo/discussions/267

谢谢！

**Comment by HEIGE-PCloud** [comment, marked as answer]
将 script 

```html
<script>window.TWIKOO_MAGIC_PATH="评论区名称"</script>
```

修改为

```html
<script>
window.TWIKOO_MAGIC_PATH="评论区名称"
document.addEventListener('pjax:send', function () {
    window.TWIKOO_MAGIC_PATH=""
});
</script>

```

**Reply by ghost** [reply]
谢谢！
